test_conformance/common/vulkan_wrapper/vulkan_api_list.hpp

```diff
@@ -89,6 +90,7 @@
     VK_FUNC_DECL(vkGetPhysicalDeviceFeatures)                                  \
     VK_FUNC_DECL(vkGetPhysicalDeviceProperties2KHR)                            \
     VK_FUNC_DECL(vkGetBufferMemoryRequirements)                                \
+    VK_FUNC_DECL(vkGetBufferMemoryRequirements2KHR)                            \
```

Comment:
Any reason we can not use core version of these APIs (vkGetBufferMemoryRequirements2) and need to use KHR extension version? Are we assuming any particular minimum Vk version ?
May be we can check for VkVersion supported and then choose between core vs KHR version of these APIs.
Also, for KHR version usage, we will need to enable the required extension during VkDevice creation.

Reply:
The only reason I used KHR was because the automated testing failed to link on Ubuntu 20.02.  I suspect the packaged Vulkan version is older than on the other platforms, however I have no way to debug github's build server.

Reply:
As far as I can see, https://github.com/KhronosGroup/OpenCL-CTS/blob/main/presubmit.sh uses Vulkan Loader and Headers from GitHub TOT. So, not sure why the automatic build may have failed while using the newer APIs.
Do you have link to the failed logs?

Reply:
Unfortunately I do not have the logs, I updated with a force push.